drheader/core.py

```diff
@@ -226,15 +226,15 @@ def __validate_must_avoid(self, config, header, directive):
                     indices = [list(policy.values()).index(item) for item in non_compliant_values]
                     for index in indices:
                         self.__add_report_item(
-                            severity='medium',
+                            severity=config['Severity'] if ("Severity" in config) else "medium",
                             error_type=5,
                             header=header,
                             directive=list(policy.keys())[index],
                             avoid=config['Must-Avoid'],
                             value=avoid_value)
                 else:
                     self.__add_report_item(
-                        severity='medium',
+                        severity=config['Severity'] if ("Severity" in config) else "high",
```

Comment:
Change the default severity back to medium
```suggestion
                        severity=config['Severity'] if ("Severity" in config) else "medium",
```